alpheios_nemo_ui/data/assets/js/infinite-scroll.js

```diff
@@ -0,0 +1,10 @@
+function preloadNext() {
```

Comment:
is this the start of work on issue #27  ? Looks like it's just debugging for now so maybe should be excluded from this PR?

Reply:
Sorry!
yes - I have started to work on infinite-scroll - and it shouldn't be here!
I will remove!